sway/ipc-json.c

```diff
@@ -578,9 +578,13 @@ static void ipc_json_describe_view(struct sway_container *c, json_object *object
 	bool visible = view_is_visible(c->view);
 	json_object_object_add(object, "visible", json_object_new_boolean(visible));
 
+	enum sway_container_layout parent_layout = container_parent_layout(c);
+	int siblings = container_get_siblings(c)->length;
+	bool has_titlebar = c->current.border != B_PIXEL
+		|| ((parent_layout == L_STACKED || parent_layout == L_TABBED) && siblings > 1);
```

Comment:
Won't stacked/tabbed containers with 1 sibling also have a title bar? Why do we only care if there are more than 1?

Reply:
Good catch. I use `hide_edge_borders --i3 none` and forgot about sway's default behavior here.

Reply:
Should be taken care of.